test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
+  for (const [shortname, json] of Object.entries(all)) {
+    describe(`The ${shortname} entry in @webidl/css`, () => {
+      for (const { type, prop, value } of cssValues) {
+        if (!json[prop]) {
+          continue;
+        }
+        for (const [name, desc] of Object.entries(json[prop])) {
+          if (!desc[value]) {
+            continue;
+          }
+          it(`defines a valid ${type} "${name}"`, () => {
+            const ast = definitionSyntax.parse(desc[value]);
```

Comment:
I guess not throwing an exception is passing. `assert.doesNotThrow` would make that more explicit. If you do that, and add an assertion message including `type`, `name` and `desc[value]`, you might be able to have just a single test per shortname here and still make failures easy to diagnose. It's not very important, but I found the very large number of tests a lot to scroll through :)